+adi/+AD4020/Rx.m

```diff
@@ -0,0 +1,116 @@
+classdef Rx < adi.common.Rx & matlabshared.libiio.base & adi.common.Attribute
+    % AD4020 Precision ADC Class
+    % adi.AD4020.Rx Receives data from the AD4020 ADC
+    %   The adi.AD4020.Rx System object is a signal source that can receive
+    %   data from the AD4020.
+    %
+    %   rx = adi.AD4020.Rx;
+    %   rx = adi.AD4020.Rx('uri','192.168.2.1');
+    %
+    %   <a href="https://www.analog.com/media/en/technical-documentation/data-sheets/ad4020-4021-4022.pdf">AD4020 Datasheet</a>
+
+    properties (Nontunable)
```

Comment:
This property is not being made use of - can be removed.

Reply:
Removed.

Reply:
Looks like the SamplesPerFrame property was removed. That is an abstract property specified in the matlab.libiio.base, so should always be assigned a value in the driver.
I was referring to the FramesCount property as the one not being used.